.appveyor.yml

```diff
@@ -95,6 +95,30 @@ environment:
         - TOXENV: py37-piplatest-coverage
           PIP: latest
 
+        # platform.linux_distribution() is removed in Python 3.8 (bpo-28167).
+        # - TOXENV: py38-pip8.1.1
+        #  PIP: 8.1.1
```

Comment:
Can we instead of commenting-out this declare explicitly in `matrix-exclude` block?

```yaml
matrix:
   exclude:
      - TOXENV: py38-pip8.1.1
```
Should work. See https://www.appveyor.com/docs/build-configuration/#exclude-configuration-from-the-matrix

Reply:
@atugushev, it's done.

Reply:
Thanks mate! Looks perfect now.